dfget/core/uploader/peer_server_executor.go

```diff
@@ -52,7 +52,7 @@ func StartPeerServerProcess(cfg *config.Config) (port int, err error) {
 	if defaultExecutor != nil {
 		return defaultExecutor.StartPeerServerProcess(cfg)
 	}
-	return 0, fmt.Errorf("executor of peer server isn't be initiliazed")
+	return 0, fmt.Errorf("executor of peer server isn't be initialized")
```

Comment:
`executor of peer server hasn't been initialized `may be better,WDYT？

Reply:
I do think so. Could you help to fix that? @BingGuanqi

Reply:
sure. I have already fixed that